artifacttemplates/http%3A%2F%2Fopentosca.org%2Fexample%2Fapplications%2Fartifacttemplates/PetClinic_DA-w1/README.md

```diff
@@ -0,0 +1,26 @@
+# PetClinic (MySQL) WAR DA [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+> Simple Java web application connecting to a MySQL database.
+
+This Service Template demonstrates how the [PetClinic](https://github.com/spring-projects/spring-petclinic) application
```

Comment:
The description of "This Service Template" is a bit strange at the Artifact Template or is the purpose to describe the context? :D